test/delivery_methods/one_signal_test.rb

```diff
@@ -0,0 +1,82 @@
+require "test_helper"
+
+class OneSignalTest < ActiveSupport::TestCase
+  setup do
+    @delivery_method = Noticed::DeliveryMethods::OneSignal.new
+    @config = {
+      app_id: "123456",
+      api_key: "key",
+      target_channel: "push",
+      json: -> {
+        {
+          include_aliases: {
+            external_id: "1234567890"
+          },
+          contents: "Hello world"
+        }
+      }
+    }
+    set_config(@config)
+  end
+
+  test "sends push notification" do
+    stub_request(:post, "https://api.onesignal.com/notifications?c=push").with(
+      headers: {
+        "Authorization" => "key",
+        "Content-Type" => "application/json"
+      },
+      body: {
+        include_aliases: {
+          external_id: "1234567890"
+        },
+        contents: "Hello world",
+        target_channel: "push",
+        app_id: "123456"
+      }
```

Comment:
```suggestion
      }.to_json
```

This should fix that stub.